src/main/java/com/alkemy/ong/mail/EmailService.java

```diff
@@ -0,0 +1,43 @@
+package com.alkemy.ong.mail;
+
+import com.sendgrid.Method;
+import com.sendgrid.Request;
+import com.sendgrid.Response;
+import com.sendgrid.SendGrid;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.Content;
+import com.sendgrid.helpers.mail.objects.Email;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+
+@Service
+public class EmailService {
+
+    @Value("${app.sendgrid.key}")
+    private String apikey;
+
+    public void sendEmail(String email) throws IOException {
+
+        Email from = new Email("lautaroyanzon13@gmail.com");
+        String subject = "Welcome message";
```

Comment:
Please use vars subject and message as method params. E.g: `sendEmail(String email, String subject, String content)`